### PR TITLE
installation - getting-started.md

### PR DESCRIPTION
**Pull request by mahmutkaya**

### CODE REVIEW

File: content/docs/getting-started.md [hunk 1]
```diff
@@ -18,109 +18,110 @@ redirect_from:
...
+Her React sürümü kendi blog gönderisini hak etmiyor ama her sürüm için yapılan değişikliklerin ayrıntıarını [React havuzunun içindeki `CHANGELOG.md` dosyasında](https://github.com/facebook/react/blob/master/CHANGELOG.md) ve [Sürümler](https://github.com/facebook/react) sayfasında bulabilirsiniz.
 
-## Versioned Documentation {#versioned-documentation}
+## Versiyon Anlatımı {#versioned-documentation}
 
-This documentation always reflects the latest stable version of React. Since React 16, you can find older versions of the documentation [on a separate page](/versions). Note that documentation for past versions is snapshotted at the time of the release, and isn't being continuously updated.
+Bu bölüm her zaman React'ın en son kararlı versiyonunu yansıtır. React 16'dan beri, açıklamaların eski sürümlerini [ayrı bir sayfada](/versions) bulabilirsiniz. Geçmiş sürümlere ait açıklamaların yayınlanma tarihinde anlık olarak görüntülendiğini ve sürekli güncellenmediğini unutmayın.
 
-## Something Missing? {#something-missing}
+## Eksik bir şey? {#something-missing}
```
Comment (alioguzhan) [inline content/docs/getting-started.md]: what about `Bir şeyler mi eksik?`

File: content/docs/getting-started.md [hunk 2]
```diff
@@ -18,109 +18,110 @@ redirect_from:
...
+React, aşamalı olarak benimsenmesi için en baştan tasarlandı ve  **ihtiyacınız kadar az veya daha fazla React kullanabilirsiniz** React’ın tadına bakmak, basit bir HTML sayfasına biraz etkileşim katmak veya karmaşık bir React destekli uygulama başlatmak isteyip istemediğinize göre, bu bölümdeki bağlantılar başlamanıza yardımcı olacaktır.
 
-### Online Playgrounds {#online-playgrounds}
+### Çevrimiçi Takılma Alanları {#online-playgrounds}
 
-If you're interested in playing around with React, you can use an online code playground. Try a Hello World template on [CodePen](codepen://hello-world), [CodeSandbox](https://codesandbox.io/s/new), or [Glitch](https://glitch.com/edit/#!/remix/starter-react-template).
+React ile takılmak istiyorsanız, çevrimiçi bir kod oyun alanını kullanabilirsiniz. Bir Merhaba Dünya şablonu deneyin [CodePen](codepen://hello-world), [CodeSandbox](https://codesandbox.io/s/new), veya [Glitch](https://glitch.com/edit/#!/remix/starter-react-template).
```
Comment (alioguzhan) [inline content/docs/getting-started.md]: `vakit gecirmek` or `oyalanmak` MAY be a better choice for `playing around`. `Takilmak` is more like `hanging out`. What do you think?

File: content/docs/getting-started.md [hunk 3]
```diff
@@ -18,109 +18,110 @@ redirect_from:
...
-People come to React from different backgrounds and with different learning styles. Whether you prefer a more theoretical or a practical approach, we hope you'll find this section helpful.
+İnsanlar farklı geçmişlerden ve farklı öğrenme tarzlarıyla React'a geliyorlar. İster teorik ister pratik bir yaklaşım tercih edersiniz, bu bölümü faydalı bulacağınızı umarız.
 
-* If you prefer to **learn by doing**, start with our [practical tutorial](/tutorial/tutorial.html).
-* If you prefer to **learn concepts step by step**, start with our [guide to main concepts](/docs/hello-world.html).
+* **Yaparak öğrenmeyi** tercih ediyorsanız, [pratik eğitim](/tutorial/tutorial.html)imizle başlayın.
+* **Kavramları adım adım öğrenmeyi** tercih ediyorsanız, [ana kavramlar rehberi](/docs/hello-world.html)mizle başlayın.
 
-Like any unfamiliar technology, React does have a learning curve. With practice and some patience, you *will* get the hang of it.
+Bilinmeyen herhangi bir teknoloji gibi, React da bir öğrenme eğrisine sahiptir. Pratik ve biraz sabır ile üstesinden *geleceksin*.
```
Comment (alioguzhan) [inline content/docs/getting-started.md]: for the integrity, I think this can / should be `geleceksiniz`

**Comment by ozcanzaferayan** [comment]
Duplicate of #26

**Pull request closed without merging**